lib/parsers/index.ts

```diff
@@ -92,3 +99,22 @@ export function createPkgTreeFromDep(dep: Dep): PkgTree {
 
   return pkgTree;
 }
+
+export function getYarnWorkSpaces(targetFile: string): YarnWorkSpaceResponse {
```

Comment:
I'm not sure we need `isYarnWorkSpace`, can't we just check `workspaces.length`?
```suggestion
export function getYarnWorkSpaces(targetFile: string): YarnWorkSpaceResponse {
  try {
    const packageJson: ManifestFile = parseManifestFile(targetFile);
    const workspaces: string[] = !!packageJson.workspaces && !!packageJson.private ? ...packageJson.workspaces : [];
    return workspaces;
  } catch (e) {
    throw new InvalidUserInputError('package.json parsing failed with ' +
      `error ${e.message}`);
  }
}
```